src/app/schema/discordUsers.json

```diff
@@ -23,26 +23,23 @@
         "bsonType": ["boolean"],
         "description": "True if discord member is a premium DEGEN pro subscriber, false otherwise"
       },
-      "ethWalletSettings": {
-        "bsonType": ["null", "object"],
+      "connectedAddresses": {
+        "bsonType": ["null", "array"],
         "description": "Wallet details.",
         "properties": {
-          "publicAddress": {
-            "bsonType": ["array"],
-            "description": "List of Ethereum public wallets.",
+            "bsonType": ["null", "string"],
+            "description": "Public addresses.",
             "minItems": 0,
-            "uniqueItems": true,
-            "items": {
-              "bsonType": ["string"],
-              "description": "Public Ethereum address."
-            }
+            "uniqueItems": true
           },
-          "isPOAPDeliveryEnabled": {
-            "bsonType": ["boolean", "null"],
-            "description": "true if user opted to have POAPs directly delivered."
+          "chainId": {
```

Comment:
We won't need to keep track of chainId per user as the chainId is the same for every user who are on the same blockchain. Instead, let's create a constant variable of all the chain Ids, here is an example https://github.com/scaffold-eth/scaffold-eth/blob/master/packages/hardhat/hardhat.config.js#L241

Reply:
Giving conditional approval.

Might need to relook at this to make sure duplicate values are not being stored

Reply:
@jonvaljonathan, once this is approved, will you make a note to follow up with @amaredeus to verify that duplicate values aren't being stored in the db?

Reply:
Yes. We will revisit in the WC PR.